.gitlab-ci.yml

```
@@ -55,13 +55,15 @@ build: &build
       when: never
     - when: on_success
   script:
-    - GRADLE_OPTS="-Dorg.gradle.jvmargs='-Xmx1900M -Xms512M' -Ddatadog.forkedMaxHeapSize=512M -Ddatadog.forkedMinHeapSize=128M" ./gradlew clean :dd-java-agent:shadowJar --build-cache --parallel --stacktrace --no-daemon --max-workers=8
+    - GRADLE_OPTS="-Dorg.gradle.jvmargs='-Xmx1900M -Xms512M' -Ddatadog.forkedMaxHeapSize=512M -Ddatadog.forkedMinHeapSize=128M" ./gradlew clean :dd-java-agent:shadowJar :dd-trace-api:jar :dd-trace-ot:jar --build-cache --parallel --stacktrace --no-daemon --max-workers=8
```

```suggestion
    - GRADLE_OPTS="-Dorg.gradle.jvmargs='-Xmx1900M -Xms512M' -Ddatadog.forkedMaxHeapSize=512M -Ddatadog.forkedMinHeapSize=128M" ./gradlew clean :dd-java-agent:shadowJar :dd-trace-api:jar :dd-trace-ot:shadowJar --build-cache --parallel --stacktrace --no-daemon --max-workers=8
```

I think `shadowJar` is required for dd-trace-ot if you intend to get the final artifact. Otherwise, only `dd-trace-ot-1.30.0-SNAPSHOT-unbundled.jar` will be generated, not `dd-trace-ot-1.30.0-SNAPSHOT.jar`.